tutorials/fundamentals/installation.md

```diff
@@ -1,30 +1,47 @@
-# napari installation tutorial
+# How to install napari on your machine
 
-Welcome to the **napari** installation tutorial!
+Welcome to the **napari** installation guide!
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
-for more advanced installation procedures. At the end of the tutorial you should
-have napari successfully installed on your computer and be able to make the
-napari viewer appear.
+This guide will teach you how to do a clean install of **napari** and launch the viewer.
+
+## Prerequisites
+
+Prerequisites differ depending on what type of distribution you want to install.
+
+### Installing the bundled app
+This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
+This installation method does not have any prerequisites. 
+[Click here](#installing-as-a-bundled-app) to see instructions
+for installing the bundled app.
```

Comment:
Possibly we should talk about the restrictions involved with choosing the bundled app over the other methods

Reply:
@GenevieveBuckley I added an important note above:

```{important}
Note that the bundled app is still
in active development, and may not be very stable. We strongly recommend installing as a Python
package instead.
```

Is this sufficient mention of the restrictions?

Reply:
Yes, I think so. Might be good to link to the python installation section/instructions in that sentence, too.